docs/changelog/3_x.rst

```diff
@@ -0,0 +1,517 @@
+  * - `.NET <https://github.com/quinchs/EdgeDB.Net>`_ (community-maintained)
+    - ``EdgeDB.Net.Driver@0.3.0``
+  * - `Elixir <https://github.com/nsidnev/edgedb-elixir>`_
+      (community-maintained)
+    - ``edgedb@0.4.0``
+
+New features
+============
+
+Simplified SDL syntax
+---------------------
+
+As part of our commitment to delivering the best developer experience in
+databases, we've made our schema definition language (or SDL) easier to use.
+You're no longer required to use the ``property`` or ``link`` keywords for
+non-computed properties and links. Also, we've replaced arrows with colons for
+a cleaner look that's easier to type.
+
+That means that this type definition:
+
+.. code-block:: sdl
+
+    type User {
+      required property email -> str;
+      multi link friends -> User;
+    }
+
+could be replaced with this equivalent one in EdgeDB 3+:
+
+.. code-block:: sdl
+
+    type User {
+      required email: str;
+      multi friends: User;
+    }
+
+As you're reading through our docs, you'll notice that changing the version
+selector in the sidebar will also switch all SDL examples to the appropriate
+SDL syntax.
+
+.. note::
+
+    SDL syntax in documentation does not change for non-versioned content like
+    guides.
+
+
+Triggers
+--------
+
+Triggers is one of the most anticipated 3.0 features! They allow you to define
+an expression to be executed whenever a given query type is run on an object
+type. The original query will *trigger* your pre-defined expression to run in a
+transaction along with the original query. These can be defined in your schema.
+
+.. code-block:: sdl
+
+    type Person {
+      required name: str;
+
+      trigger log_insert after insert for each do (
+        insert Log {
+          action := 'insert',
+          target_name := __new__.name
+        }
+      );
+    }
+
+The trigger above inserts a ``Log`` object any time a ``Person`` object is
+inserted.
+
+
+Mutation rewrites
+-----------------
+
+The mutation rewrites feature is the sibling, or at least the first cousin, of
+triggers. Both are automatically invoked when a write operation occurs on the
+type they're on, but triggers are not able to make changes to the object that
+invoked them. Mutation rewrites are built to do just that!
+
+.. code-block:: sdl
+
+    type Post {
+      required title: str;
+      required body: str;
+      modified: datetime {
+        rewrite insert, update using (datetime_of_statement())
+      }
+    }
+
+This shows one reason mutation rewrites is one of our most wanted features:
+modified timestamps! When a user inserts or updates a ``Post``, the rewrite
+will set the value of the ``modified`` property to that value of
+``datetime_of_statement()``. There are tons of other uses too. Give them a try!
+
+
+Splats
+------
+
+This is one of the most fun features in 3.0, both to say _and_ to use! With
+splats, you can easily select all properties in your queries without typing all
+of them out.
```

Comment:
Would be nice to show just how awesome the splat operator is with a before/after comparison so new readers are fully aware. Something like this:

What you had to type before: 
select Movie { id, release_year, title };

Now:
select Movie {*};

What you had to type before:
select Movie { 
  id, 
  release_year,
  title,
  characters: {
    id,
    name
  }
}

Now:
select Movie {**};